Src/Support/IntegrationTests/IntegrationTests.csproj

```diff
@@ -25,4 +25,28 @@
     <Reference Include="System.Net.Http" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+    <PackageReference Include="Microsoft.Extensions.Hosting">
```

Comment:
These would all be a lot more compact using attributes instead of elements for the versions - and I suspect you can use one combined condition instead of two separate ItemGroups.

Reply:
Ah yes, done, just let Visual Studio add it for me before...